src/jira/client/jira-api-client.ts

```diff
@@ -0,0 +1,494 @@
+import Logger from "bunyan";
```

Comment:
Sorry its massive, mostly just brought things over from old clients as is, will highlight with comments changes